### PR TITLE
Fix requirements

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,6 +49,9 @@ requirements:
     - python
     - czmq
     - {{ mpi }}
+    - cffi
+    - ply
+    - lua-luaposix
```
Comment (grondo) [inline recipe/meta.yaml:54]: Can you clarify the difference between `host` and `run`? e.g. also libarchive, libhwloc, sqlite, zeromq, etc are required at runtime if this what the `run` tag indicates.

I good test would be to install this flux-core package if possible and try `flux start -s 4 flux run -N4 hostname` which exercises not all but many parts of the code, including the Python requirements.

**Review by grondo — COMMENTED** [review]
I'm unsure if more needs to be added to the `run` section here (pretty much all the `host` requirements except `valgrind` and `sphinx` are required at runtime. (And I apologize if I'm still not understanding conda)

File: recipe/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipe/meta.yaml:54]: Regarding the `host` vs `run`: https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-host-and-run 
Still it also depends on the `run_exports` in the other packages, if those are missing for `shared libraries` then we have to add them as well. I try to push those changes up-stream but that might take some time, so in the mean time I added them here. In terms of the tests, that is something I plan to do at the end of the up-coming week. It is all a bit busy right now.

Comment (grondo) [reply recipe/meta.yaml:54]: Ah, ok, that is starting to make sense. Thanks!

**Review by grondo — APPROVED** [review]
LGTM

**Pull request merged**